aries-site/src/pages/whats-new.mdx

```diff
@@ -51,9 +51,17 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 --><!-- END: New Entry Template -->
 
 <!-- BEGIN: What's New Entries -->
+## July 26 - August 31, 2021 
+- Published the [Notification](https://design-system.hpe.design/components#visualizations) and [Toast Notifications](https://design-system.hpe.design/templates/toast-notifications) & [Status Indicator](https://design-system.hpe.design/templates/status-indicator) content with examples. 
+- Updated [toolbar button to include an icon only variant + changed stroke from 2px to 1px](https://www.figma.com/file/Oi5UEEQ33VCAyOKQcZ8Nr8/HPE-Button-Component?node-id=0%3A1) 
```

Comment:
Capitalization consistency between "Toolbar Button" and "Default Button" listed below.

End sentences with period.

Trying to not hyperlink the majority of the sentence, focussing on the subject.

```suggestion
- Updated [Toolbar Button](https://www.figma.com/file/Oi5UEEQ33VCAyOKQcZ8Nr8/HPE-Button-Component?node-id=0%3A1)  to include an icon only variant + changed stroke from 2px to 1px.
```